app/views/idv/sessions/step1.html.slim

```diff
@@ -0,0 +1,48 @@
+- title t('idv.titles.welcome')
```

Comment:
since this really the second step, should we maybe just name this template `update.html.slim` so that it matches the controller method and there is no need for an explicit call to `render` ?